src/components/Common/SEO.js

```diff
@@ -1,12 +1,11 @@
 import { Helmet } from "react-helmet";
 
-import { t } from "@lingui/macro";
-
 function SEO(props) {
   const { children, ...customMeta } = props;
   const meta = {
-    title: t`GMX | Decentralized Perpetual Exchange`,
-    description: t`Trade spot or perpetual BTC, ETH, AVAX and other top cryptocurrencies with up to 30x leverage directly from your wallet on Arbitrum and Avalanche.`,
```

Comment:
hm, sorry, it's ok that we remove `t` from here? Then the title and description will be in English only?